python/ray/data/_internal/compute.py

```diff
@@ -44,13 +45,17 @@ def __init__(
             raise ValueError("`size` must be >= 1", size)
         self.size = size
 
+    def dataset_export_args(self) -> Dict[str, Any]:
+        """Export the arguments into dictionary format."""
+        return {"size": self.size}
```

Comment:
DatasetExportArgs is meant for operators. For this class, I think it's fine to just make `__str__` representative.